pkg/api-server/service_insight_endpoints.go

```diff
@@ -135,7 +144,12 @@ func (s *serviceInsightEndpoints) fillStaticInfo(name string, stat *v1alpha1.Ser
 // 2) Mesh+Name is a key on Universal, but not on Kubernetes, so if there are two services of the same name in different Meshes we would have problems with naming.
 // From the API perspective it's better to provide ServiceInsight per Service, not per Mesh.
 // For this reason, this method expand the one ServiceInsight resource for the mesh to resource per service
-func (s *serviceInsightEndpoints) expandInsights(serviceInsightList *mesh.ServiceInsightResourceList, nameContains string, filterFn func(service *v1alpha1.ServiceInsight_Service) bool) []rest.Resource {
+func (s *serviceInsightEndpoints) expandInsights(
+	serviceInsightList *mesh.ServiceInsightResourceList,
+	nameContains string,
+	filterFn func(service *v1alpha1.ServiceInsight_Service) bool,
+	mapperFn unversionedResourceMapper,
```

Comment:
why do we need a mapper function here? Do we expect other implementations? If not, can't we just call removeDisplayNameLabel directly?

Reply:
I was just wondering if we need other mapper implementations in the further, tha's why I write a `mapperFn` here. Surely we can just call `removeDisplayNameLabel` directly, let me do that.